### PR TITLE
Deactivate filename truncation

### PR DESCRIPTION
**Pull request by liry**

### CODE REVIEW

**Comment by roblav96** [comment]
@liry No pics no clicks 😛 Can you show a screenshot of what these modifications do?

File: manifest.json [hunk 1]
```diff
@@ -17,11 +17,5 @@
   ],
   "background": {
     "scripts": ["background.js"]
-  },
-  "licenses": [
```
Comment (Martii) [inline manifest.json]: Removal of this section will break building and possibly unlicense some of the output.

File: manifest.json [hunk 2]
```diff
@@ -2,8 +2,8 @@
   "manifest_version": 2,
 
   "name": "Wide Github",
-  "description": "Change all Github repository and gist pages to be full width and dynamically sized.",
-  "version": "1.4.0",
+  "description": "Change all Github repository and gist pages to be full width and dynamically sized. Also deactivates filename truncation.",
```
Comment (Martii) [inline manifest.json]: As someone else mentioned a screenshot with URL would be appreciated to what this means.

**Comment by liry** [comment]
Well, in a meantime, GitHub changed their implementation and it's not truncating filenames using CSS, so it doesn't work now anyway :(

**Pull request closed without merging**